### DISCUSSION TITLE
Hotspot Owner

### DISCUSSION THREAD

**Discussion by jobersomething** [davetapley/helium-tax#23] [Q&A] [answered]
This tool is so awesome. Thank you! I would love to be able to use my Helium owner ID to generate the exact same report for multiple hotspots. Possible?

**Comment by davetapley** [comment, marked as answer]
~Yes! It's actually already been requested on https://github.com/davetapley/helium-tax/issues/4.~

~I'm not sure when I'll get to it, but you're welcome to 'subscribe' to that issue (#4) and you'll get an email if/when I get to it~

This is done and live! 💫

**Comment by jobersomething** [comment]
Awesome! Thanks!
